.travis.yml

```
@@ -33,6 +37,7 @@ jobs:
       before_script: npm run lint
       script:
         - npm run build --prod
+        - docker run --rm -d -p 7770-7777:7770-7777 --name conode c4dt/conode:dev
```

```suggestion
        - make docker_run &
```

---

I think this will not work, as this is in the `webapp` directory, and this one has no `docker_run` target.

---

Ho, you're right, I was a bit quick
```suggestion
        - make -C ../conode docker_run &
```